src/utils/error_checking.cpp

```diff
@@ -0,0 +1,46 @@
+//========================================================================================
+// Athena++ astrophysical MHD code
+// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+// Licensed under the 3-clause BSD License, see LICENSE file for details
```

Comment:
This code was not taken from Athena++, wasn't it? So 
```suggestion
// Parthenon performance portable AMR framework
// Copyright(C) 2020 The Parthenon collaboration
// Licensed under the 3-clause BSD License, see LICENSE file for details
```
may be more appropriate, wouldn't it?

Reply:
Yes, though I don't think error_checking.hpp was either...